packages/suite/src/components/suite/PasswordStrengthIndicator/index.tsx

```diff
@@ -57,20 +59,31 @@ interface Props {
 }
 
 const PasswordStrengthIndicator = ({ password }: Props) => {
-    const [score, setScore] = useState<ZXCVBNScore>(0);
+    const [score, setScore] = useState<OptionalZXCVBNScore>();
     useEffect(() => {
         const runScoring = async () => {
-            const pwScore = await getPasswordScore(password);
-            setScore(pwScore);
+            try {
+                const pwScore = await getPasswordScore(password);
+                setScore(pwScore);
+            } catch (error) {
+                //
```

Comment:
Why not to report the error at least via `console.error(error);`? Please add reporting or add explanation comment why we ignore it here...

btw, it's no more necessary to define `(error)` if you don't use it (see "Optional catch Binding").

Reply:
Done, squashed, rebased.